pubspec.yaml

```diff
@@ -47,8 +47,10 @@ dependencies:
   path: 1.8.1
   provider: 6.0.3
   get_it: 7.2.0
```

Comment:
It is a good pattern to sort lines alphabetically, in this case it can be lines 41 - 50. To do it you can use `Ctrl + Shift + A` and then 
![image](https://user-images.githubusercontent.com/6877292/175861266-1c4c14a9-e0f5-48cf-ab51-bf90223b1845.png)

Reply:
do this. but i got error because wrong order of string. and i think all strings in mess.

Reply:
done